src/shared/HtmlReady.test.js

```diff
@@ -82,4 +82,11 @@ describe('htmlready', () => {
         const res = HtmlReady(somanylinks).html;
         expect(res).to.equal(htmlified);
     });
+
+    it('should link usernames', () => {
+        const textwithmentions = '<xml xmlns="http://www.w3.org/1999/xhtml">@username (@a1b2, whatever</xml>';
+        const htmlified = '<xml xmlns="http://www.w3.org/1999/xhtml"><span><a href="/@username">@username</a> (<a href="/@a1b2">@a1b2</a>, whatever</span></xml>';
+        const res = HtmlReady(textwithmentions).html;
```

Comment:
Is there a reason it tests the html output and not simply the output state of `usertags`?

Reply:
i wrote the tests this way because it gives a clear end-user picture of what htmlready does -- we should have tests covering `linkifyNode` and `linkify` specifically, also